### ISSUE TITLE
Ability to preview and edit/save WP before loading into NavGrid as target.

### ISSUE THREAD

**Issue by vinteo** [pcbennion/starbase-navgrid#17]
Been testing NavGrid for a few days in the PTU on my new ship and its working great. 

However there is one thing about the waypoint system that I feel should work differently. And that is to have a separate step to "load" a WP to be used. That way moving up/down on the waypoint does not immediately change the display of the NavGrid and we can view/edit/save a WP that is different from the one that is targeted on NavGrid.

This may not affect NavGrid as much but if I wanted to integrate your WP system with an autopilot and use :nx, :ny and :nz as the target, once I engage the autopilot I cannot view or edit any of the other WPs  cause as soon as I change to another WP the target changes.

Of course the other option for me is to do my own "load" to the autopilot which I might do if you feel like this is not something you want to do for your WP system. But I would prefer if I could use the single "target" for NavGrid and autopilot. And I could use a different WP system as well but I really like your system especially around the WP naming which I feel works the best among the ones I have used before.

I hope I explained the issue well enough and its not too confusing.

PS. I know this will probably require more memory but I feel like its worth it. The expansion does not need to ne optional and we can use the extra memory space there for :wx :wy and :wz

**Comment by pcbennion** [comment]
I think I understand what you're getting at. The NavP and nName/nX/nY/nZ displays could be more of a 'waypoint browser' and you'd lock in your NavGrid destination with a separate button. It'd be useful if the waypoint computer didn't lock up during long-haul trips.

The navball feature will require more chips anyway so I can definitely find space for it. I'll set this as a 2.0 feature so I can ask around for design feedback and see what other people think.

**Comment by vinteo** [comment]
yes exactly , you explained it in 2 sentences and I wrote an essay haha.
